create_framework/http-kernel-httpkernelinterface.rst

```diff
@@ -153,7 +153,7 @@ sub-requests to convert them to their proper content::
     $framework = new HttpKernel\HttpCache\HttpCache(
         $framework,
         new HttpKernel\HttpCache\Store(__DIR__.'/../cache'),
-        new HttpKernel\HttpCache\ESI()
+        new HttpKernel\HttpCache\Esi()
```

Comment:
Shouldn't we also fix the namespace (the `Symfony\Component` part is missing and there is no `use` statement for it)?